### DISCUSSION TITLE
점프시에 사운드를 따로 넣으려면 어떻게 해야 할까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#579] [5. Q&A]
현재 UI를 패키지에서 에셋으로 받아와 새로운 스크립트를 추가해서 Jump버튼에 할당하는 식으로 변경했습니다.

허나 이렇게하면 점프버튼을 누를때마다 사운드가 재생되더군요...

혹시 제페토 플레이어의 점프 애니메이션이 재생되는 거를 파악할 수 있을까요? 

아니면 플레이어가 점프 가능/불가능한 상태를 확인할 수 있는 방법이라도 좋습니다!

**Comment by pumfit** [comment]
캐릭터의 점프 상태를 아래와 같이 CharacterState를  통해 파악할 수 있어서 아래와 같은 조건문으로 처리하시면 될 것 같습니다!
```
const character = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;

if(character.CurrentState === CharacterState.JumpIdle || character.CurrentState === CharacterState.JumpMove)
{
...점프 시 관련 처리
}
```

**Reply by ted002** [reply]
오호 감사합니다!
